distribution/docs/src/main/resources/_contents/_apis/migration-api-contents.adoc

```diff
@@ -0,0 +1,104 @@
+
+=== Migration API
+
+[NOTE]
+====
+This code is experimental. While this interface is functional and tested, it may change or be removed in a future version of the library.
+====
+
+${branding} currently has an experimental API for making bundles migratable. Interfaces in `platform/migration/platform-migratable-api` are
+used by the system to identify bundles that provide implementations for import (_coming soon_) and export operations.
```

Comment:
I personally would switch import and export in this sentence as they are by necessity run in that order (and import isn't finished yet).